Python/Python.sublime-syntax

```diff
@@ -402,6 +402,120 @@ contexts:
       pop: true
 
   comments:
+    # Type-ignore can only be used by itself.
+    - match: '(#)\s*(type)(:)\s*(ignore)\b'
+      scope: comment.line.type-hint.python
+      captures:
+        1: punctuation.definition.comment.python comment.line.number-sign.python
+        2: keyword.other.annotation.type-comment.python
+        3: punctuation.separator.annotation.type-comment.python
+        4: keyword.other.annotation.type-comment.ignore.python
+      set: comments_afterignore
+    - match: '(#)\s*(type)(:)'
+      scope: comment.line.type-hint.python
+      captures:
+        1: punctuation.definition.comment.python comment.line.number-sign.python
+        2: keyword.other.annotation.type-comment.python
+        3: punctuation.separator.annotation.type-comment.python
+      embed: type-comment
+      escape: '\n'
+      embed_scope: comment.line.type-hint.python
+
+    - include: comments_noannotation
+
+  comments_afterignore:
```

Comment:
It seems stylistically that other context named use `-` as a separator, not `_`. Let's stay consistent with that.